assembl/static2/js/app/components/administration/voteSession/modulesSection.jsx

```diff
@@ -26,17 +47,55 @@ const DumbModulesSection = ({ tokenModules, editLocale, handleCheckBoxChange }:
             <Checkbox
               checked={tokenModuleChecked}
               onChange={() => {
-                handleCheckBoxChange(tokenModuleChecked, tModule[0]);
+                handleTokenCheckBoxChange(tokenModuleChecked, tModule[0], newId);
               }}
             >
               <Helper
                 label={I18n.t('administration.voteWithTokens')}
                 helperUrl="/static2/img/helpers/helper4.png"
                 helperText={I18n.t('administration.tokenVoteCheckbox')}
-                classname="inline"
+                classname="inline checkbox-title"
               />
             </Checkbox>
             {tokenModules.map(id => <TokensForm key={id} id={id} editLocale={editLocale} />)}
+            <Checkbox
+              checked={gaugeModuleChecked}
+              onChange={() => {
+                handleGaugeCheckBoxChange(gaugeModuleChecked, gModule, newId);
+              }}
+            >
+              <Helper
+                label={I18n.t('administration.voteWithGages')}
+                helperUrl="/static2/img/helpers/helper3.png" // TODO: add an actual screenshot
+                helperText={I18n.t('administration.gaugeVoteCheckbox')}
+                classname="inline checkbox-title"
+              />
+            </Checkbox>
+            {gaugeModuleChecked ? (
+              <div>
+                <div className="flex">
+                  <label htmlFor="input-dropdown-addon">
+                    <Translate value="administration.gaugeNumber" />
+                  </label>
+                  <Helper helperUrl="/static2/img/helpers/helper2.png" helperText={I18n.t('administration.defineGaugeNumer')} />
+                </div>
+                <SplitButton
+                  title={gaugeModules.size}
+                  id="input-dropdown-addon"
+                  required
+                  onSelect={(eventKey) => {
+                    handleGaugeSelectChange(eventKey, gaugeModules.size, newId, gModule);
+                  }}
+                >
+                  {range(10).map(value => (
```

Comment:
is that really `10` here? shouldn't it be the number of "steps" configured for the gauge?